### PR TITLE
75 create seeder

### PR DESCRIPTION
**Pull request by rizalsidikp**
#75 

### Breakdown:
Add seeder for the followings:
- [x] events - already there, but need more exxmple data
- [x] beacons
- [x] users
- [ ] spots (Mr. @andy-shi88 will handle this)
- [x] stages
- [x] schedules

### CODE REVIEW

File: seeders/users_seeder.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            user = {
+                'first_name': first_name,
+                'last_name': last_name,
+                'email': email,
+                'username': username,
+                'password': password,
+                'role_id': role_id
+            }   
+            new_user = User()
+            new_user.first_name = user['first_name']
```
Comment (andy-shi88) [inline seeders/users_seeder.py]: assign directly with `new_user.first_name = fake.first_name()`, the other can be done the same way

File: seeders/stages_seeder.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            stage_type = random.choice(stage_types)
+            information = fake.sentence(
+                nb_words=6, variable_nb_words=True, ext_word_list=None)
+            stage = {
+                'name': name,
+                'stage_type': stage_type,
+                'information': information
+            }     
+            new_stage = Stage()
+            new_stage.name = stage['name']
```
Comment (andy-shi88) [inline seeders/stages_seeder.py]: the value can be assign directly you don't need to create object first.

File: seeders/schedules_seeder.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            time_end = time_start + timedelta(hours=rand_int)
+            schedule = {
+                'user_id': user_id,
+                'event_id': event_id,
+                'stage_id': stage_id,
+                'time_start': time_start,
+                'time_end': time_end
+            }
+            new_schedule = Schedule()
+            new_schedule.user_id = schedule['user_id']
```
Comment (andy-shi88) [inline seeders/schedules_seeder.py]: assign directly, look at the comment on the user seeder.

File: seeders/users_seeder.py (hunk 1 above)
Comment (rizalsidikp) [inline seeders/users_seeder.py]: done

File: seeders/schedules_seeder.py (hunk 3 above)
Comment (rizalsidikp) [inline seeders/schedules_seeder.py]: this is done too

**Comment by andy-shi88** [comment]
thanks @rizalsidikp

**Comment by rizalsidikp** [comment]
okayy @andy-shi88

**Pull request merged**